lib/src/animated_map_controller.dart

```diff
@@ -134,7 +134,8 @@ class AnimatedMapController {
     final bool hasRotation = rotation != null && rotation != this.rotation;
     final bool hasMovement =
         (dest != null && dest != mapController.camera.center) ||
-            (zoom != null && zoom != mapController.camera.zoom);
+            (zoom != null && zoom != mapController.camera.zoom) ||
+            (offset != null);
```

Comment:
There's an issue with the analyzer, it should be something similar to this:
```dart
offset != Offset.zero
```

Reply:
Got it. Sounds good to me!